connectors/byoei.py

```diff
@@ -167,25 +177,30 @@ async def run(self):
 class Fetcher:
     """Grabs data and adds them in the queue for the bulker.
 
-    This class runs a coroutine that puts docs in `queue`, given
-    a document generator.
+    This class runs a coroutine that puts docs in `queue`, given a document generator.
+
+    Arguments:
+    - queue: an `asyncio.Queue` to put docs in
+    - index: the target Elasticsearch index
+    - existing_ids: a list of existing ELasticsearch documents ids found in the index
+    - filter: a `Filter` instance -- default: `None`
```

Comment:
Similar to other comment;
```suggestion
    - filter: an instance of `Filter` -- default: `None`
```

Reply:
```suggestion
    - filter_: an instance of `Filter` to apply on the fetched document -- default: `None`
```